Grammar/python.gram

```diff
@@ -1230,6 +1233,10 @@ invalid_group:
         RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "cannot use starred expression here") }
     | '(' a='**' expression ')' {
         RAISE_SYNTAX_ERROR_KNOWN_LOCATION(a, "cannot use double starred expression here") }
+invalid_import:
+    | a='import' dotted_name 'from' dotted_name {
+        RAISE_SYNTAX_ERROR_STARTING_FROM(a, "Did you meant to use 'from ... import ...' instead?") }
```

Comment:
Ditto in tests and docs.
```suggestion
        RAISE_SYNTAX_ERROR_STARTING_FROM(a, "Did you mean to use 'from ... import ...' instead?") }
```

Reply:
Done!

Reply:
Of course! I think you missed the one in `whatsnew`?